oathkeeper/12_ory_cloud_keto/README.md

```diff
@@ -0,0 +1,34 @@
+## An example using Ory Cloud with Ory Keto
+
+This example shows basic configuration of `cookie_session` authenticator for oathkeeper and `remote_json` for authorization
+
+## Running locally
+
+```
+   git clone git@github.com:ory/examples
+   cd examples/oathkeeper/12_ory_cloud_keto
+   docker-compose up
+```
+
+## Running Ory proxy
+
+```
+   ory proxy http://127.0.0.1:8080
+```
+
+## Configuring Ory Keto Cloud
+
+1. Edit `cli.sh` and put your personal access token and change the URL of your Ory instance
+1. Run it `bash cli.sh`
```

Comment:
Everything works fine up until here, I am getting some errors with the request:

{"error":{"code":404,"status":"Not Found","request":"34ca2b32-9fce-9c8d-8601-749605e9aab9","reason":"Unknown namespace with name services.","message":"The requested resource could not be found"}}

Do I need to create a namespace first?

Reply:
Yep. I wrote cli.sh especially for this

Reply:
I'll update it soon